src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  margin-right: $card-hor-mar;
+  margin-bottom: $card-bot-mar;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 46px;
+  }
+
+  &__img {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__caption {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+    width: 100%;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 16px;
+    font-weight: 400;
+  }
+
+  &__price-info {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  &__price-title {
+    margin-bottom: 16px;
+
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__price-price {
+    font-weight: 700;
+  }
+
+  &__buy-button {
+    display: block;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 38px;
+    text-align: center;
+    text-decoration: none;
+    color: white;
+
+    background-color: #00acdc;
```

Comment:
You often use this color in styles. it's better to add it to a variable.

Reply:
fixed